corehq/apps/accounting/models.py

```diff
@@ -964,6 +1026,10 @@ def renew_subscription(self, date_end=None, note=None, web_user=None,
             date_start=self.date_end,
             date_end=date_end,
         )
+        if type is not None:
```

Comment:
Feel free to rename "type" if it's confusing for the dev reading the code.
service_type makes sense as well.

On Wed, Feb 11, 2015 at 2:17 PM, Ben <notifications@github.com> wrote:

> In corehq/apps/accounting/models.py
> <https://github.com/dimagi/commcare-hq/pull/5500#discussion_r24523893>:
>
> > @@ -964,6 +1026,10 @@ def renew_subscription(self, date_end=None, note=None, web_user=None,
> >              date_start=self.date_end,
> >              date_end=date_end,
> >          )
> > +        if type is not None:
>
> fair, sounds good
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/5500/files#r24523893>.
>